content/guides/language/bun/_index.md

```diff
@@ -0,0 +1,27 @@
+---
+description: Containerize and develop Bun applications using Docker.
+keywords: getting started, bun
+title: Bun language-specific guide
+linkTitle: Bun
+toc_min: 1
+toc_max: 2
+aliases:
+- /language/bun/
```

Comment:
Aliases are only used to redirect pages that have moved around - this is a new page so no need for this

```suggestion
```

Reply:
Ah, got it. Thank you!